android-sdk/src/main/java/ai/evolv/android_sdk/WaitForIt.java

```diff
@@ -0,0 +1,103 @@
+package ai.evolv.android_sdk;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import ai.evolv.android_sdk.evolvinterface.EvolvAction;
+import ai.evolv.android_sdk.evolvinterface.EvolvContext;
+import ai.evolv.android_sdk.evolvinterface.EvolvInvocation;
+
+import static ai.evolv.android_sdk.EvolvContextImpl.CONTEXT_VALUE_ADDED;
+import static ai.evolv.android_sdk.EvolvContextImpl.CONTEXT_VALUE_REMOVED;
+
+class WaitForIt {
```

Comment:
Please add the TODO to get this implemented -- also, very testable this class

Reply:
sure